front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -91,6 +92,7 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
   };
 
   const onSelectSuggestion = (option?: LightRollingStockWithLiveries) => {
```

Comment:
There is still an issue here. If I clear the towed in put and then clear the rs input, this function receives the stale towed data even though the towed id is at undefined and the inputs are refilled with these datas.

To fix this, you can do in `useStdcmTowedRollingStock` the same thing we did in `useStoreDataForRollingStockSelector`, changing `data` to `currentData` in the endpoint call. This will ensure to provide an `undefined` towed if the last call returned `undefined` instead of providing the stale data.